public/app/features/users/state/selectors.ts

```diff
@@ -1,18 +1,18 @@
-export const getUsers = state => {
+export const getUsers = (state: any) => {
   const regex = new RegExp(state.searchQuery, 'i');
 
-  return state.users.filter(user => {
+  return state.users.filter((user: any) => {
     return regex.test(user.login) || regex.test(user.email);
   });
 };
 
-export const getInvitees = state => {
+export const getInvitees = (state: any) => {
   const regex = new RegExp(state.searchQuery, 'i');
 
-  return state.invitees.filter(invitee => {
+  return state.invitees.filter((invitee: any) => {
```

Comment:
is filter array function not a natively typed (generically)  in typescript ?

Reply:
Yeah, it is, I think I just went a little bit on autopilot here